src/helics/core/BrokerBase.cpp

```diff
@@ -235,7 +236,11 @@ std::shared_ptr<helicsCLI11App> BrokerBase::generateBaseCLI()
         "--dumplog",
         dumplog,
         "capture a record of all messages and dump a complete log to file or console on termination");
-
+    auto* lbs = logging_group->add_option(
+        "--logbuffersize",
+        mlogBufferSize,
+        "specify the size of the circular buffer for storing log messages for later retrieval");
+    logging_group->add_flag("--logbuffer{10}", mlogBufferSize, "specify that the ")->excludes(lbs);
```

Comment:
The help string for this flag looks like it got cut off.

Reply:
Do you think it would be clearer as just a single option?